tools/webpack-plugin/src/BacktracePlugin.ts

```diff
@@ -48,30 +52,40 @@ export class BacktracePlugin implements WebpackPluginInstance {
             for (const [asset, sourcePath, sourceMapPath] of entries) {
                 logger.time(`[${asset}] process source and sourcemap`);
 
+                const stats: AssetStats = {};
+                assetStats.set(asset, stats);
+
                 let debugId: string;
                 try {
                     debugId = await this._sourceMapProcessor.processSourceAndSourceMapFiles(sourcePath, sourceMapPath);
+                    stats.debugId = debugId;
+                    stats.processSource = true;
                     logger.timeEnd(`[${asset}] process source and sourcemap`);
                 } catch (err) {
                     logger.timeEnd(`[${asset}] process source and sourcemap`);
                     logger.error(`[${asset}] process source and sourcemap failed:`, err);
+                    stats.processSource = err instanceof Error ? err : new Error('Unknown error.');
                     continue;
                 }
 
                 if (this._sourceMapUploader) {
```

Comment:
is there any reason to upload source map without debugId?

Reply:
We do not upload a sourcemap without debugId - see `continue` on line 68.